test/evergreen_develop.yml

```diff
@@ -98,7 +98,8 @@ buildvariants:
     - name: many-dhandle-stress
     - name: ".wt2853-perf"
     - name: cppsuite-bounded-cursor-perf-stress
-    - name: many-collection-test
+    # FIXME-WT-11599 Disable until python changes are implemented.
+    # - name: many-collection-test
       distros: ubuntu2004-wt-large
```

Comment:
This should be commented out as well.

Reply:
The distros that is.

Reply:
done here [1c86b14](https://github.com/wiredtiger/wiredtiger/pull/9607/commits/1c86b1493581b1f71d32d9dba33f44fa076ad05d)